importers/src/bible/osis/index.ts

```diff
@@ -0,0 +1,945 @@
+                    contents: []
+                };
+
+                context.sectionStack.push(elementType);
+                currentContainer.contents.push(section);
+                context.contentContainerStack.push(section);
+
+                break;
+            }
+            case OsisXmlNodeName.PARAGRAPH:
+            case OsisXmlNodeType.PARAGRAPH: {
+                let currentContainer = this.getCurrentContainer(context);
+
+                const paragraph: IBibleContentGroup<'paragraph'> = {
+                    type: 'group',
+                    groupType: 'paragraph',
+                    contents: []
+                };
+
+                currentContainer.contents.push(paragraph);
+                context.contentContainerStack.push(paragraph);
+                break;
+            }
+            case OsisXmlNodeName.LINE_GROUP: {
+                let currentContainer = this.getCurrentContainer(context);
+
+                const linegroupGroup: IBibleContentGroup<'linegroup'> = {
+                    type: 'group',
+                    groupType: 'linegroup',
+                    contents: []
+                };
+
+                if (
+                    currentContainer.type !== 'group' ||
+                    currentContainer.groupType !== 'paragraph'
+                ) {
+                    const paragraph: IBibleContentGroup<'paragraph'> = {
+                        type: 'group',
+                        groupType: 'paragraph',
+                        contents: [linegroupGroup]
+                    };
+
+                    currentContainer.contents.push(paragraph);
+                } else currentContainer.contents.push(linegroupGroup);
+                context.contentContainerStack.push(linegroupGroup);
+                break;
+            }
+            case OsisXmlNodeName.LINE: {
+                let currentContainer = this.getCurrentContainer(context);
+                let lineNr =
+                    currentContainer.type === 'group' && currentContainer.groupType === 'linegroup'
+                        ? currentContainer.contents.length + 1
+                        : 1;
+                const lineGroup: IBibleContentGroup<'line'> = {
+                    type: 'group',
+                    groupType: 'line',
+                    modifier: lineNr,
+                    contents: []
+                };
+                // RADAR: currently only taking into account a second level of indentation here
+                //        (the first level is already taken care of by the linegroup)
+                if (tag.attributes.level && +tag.attributes.level > 1) {
+                    const indentGroup: IBibleContentGroup<'indent'> = {
+                        type: 'group',
+                        groupType: 'indent',
+                        contents: [lineGroup]
+                    };
+
+                    currentContainer.contents.push(indentGroup);
+                } else currentContainer.contents.push(lineGroup);
```

Comment:
prefer: https://eslint.org/docs/rules/curly

Reply:
i personally like having one-liners (if they are really short).
if we go for that change i think we would need to make it consistent project-wide. also a thing that would need a general conversation.. and i tend to disagree with eslint here.